buildroot-external/patches/linux/6.6.25/0001-ipv6-add-option-to-explicitly-enable-reachability-te.patch

```diff
@@ -1,52 +0,0 @@
-From 76591e4075194cf717dc085b8285912f706bcd46 Mon Sep 17 00:00:00 2001
-From: Stefan Agner <stefan@agner.ch>
-Date: Tue, 28 Mar 2023 12:02:10 +0200
-Subject: [PATCH] ipv6: add option to explicitly enable reachability test
```

Comment:
This patch seems to be missing now? :thinking:

Reply:
Not really, now it's been replaced by the patch all versions in `buildroot-external/patches/linux/0001-ipv6-add-option-to-explicitly-enable-reachability-te.patch`